torch/_inductor/ir.py

```diff
@@ -7067,6 +7068,8 @@ def __str__(self) -> str:
 class TensorBox(MutableBox):
     @staticmethod
     def create(data):  # type: ignore[no-untyped-def]
+        if isinstance(data, ShapeAsConstantBuffer):
```

Comment:
do you know when we create these ? just curious

Reply:
I think in `realize_input` of ExternKernel. If that input happens to be output as well, then this PR is needed to make it work.